gui/src/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+set(RIME_CONFIG_SRCS
+  Main.cpp
+  ConfigMain.cpp
+  Model.cpp
+  RimeConfigParser.cpp
+  ErrorOverlay.cpp)
+
+set(RIME_CONFIG_HDRS
+  Main.h
+  ConfigMain.h
+  Model.h
+  Common.h
+  RimeConfigParser.h
+  ErrorOverlay.h)
+
+set(RIME_CONFIG_UIS
+  ConfigMain.ui)
+
+# fcitx5_translate_add_sources(
```

Comment:
not required.

Reply:
This is done through Messages.sh in fcitx5. You'll need to check if Messages.sh handles all the files and formats. If not, you can copy-paste from some other fcitx5 repo that handles it.